.devcontainer/devcontainer.json

```diff
@@ -19,9 +19,6 @@
 	// Add the IDs of extensions you want installed when the container is created.
 	"extensions": [
 		"ms-dotnettools.csharp",
-        "ms-vscode-remote.remote-containers",
-        "ms-vscode-remote.remote-wsl",
-        "ms-azuretools.vscode-docker",
```

Comment:
Why remove these?

Reply:
inside the container they have no use (these are specs for the container environment)